pkg/function/backup_data.go

```diff
@@ -124,6 +124,11 @@ func (*backupDataFunc) RequiredArgs() []string {
 		BackupDataIncludePathArg, BackupDataBackupArtifactPrefixArg}
 }
 
+func (*backupDataFunc) Arguments() []string {
+	return []string{BackupDataNamespaceArg, BackupDataPodArg, BackupDataContainerArg,
+		BackupDataIncludePathArg, BackupDataBackupArtifactPrefixArg, BackupDataEncryptionKeyArg}
```

Comment:
```suggestion
	return []string{
		BackupDataNamespaceArg,
		BackupDataPodArg,
		BackupDataContainerArg,
		BackupDataIncludePathArg,
		BackupDataBackupArtifactPrefixArg,
		BackupDataEncryptionKeyArg,
	}
```
Separately we can do this for multiple lines of required args too